.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

Comment:
as above

Reply:
Let me know if you want to rename the branch right now or after we merge this.

Also,  if we don't rename the `master` branch now, let me know if you want to keep both mentions here.   Right now  `main` is simply ignored by the actions, it doesn't break anyting.